api/app/models/spree/webhooks/subscriber.rb

```diff
@@ -51,6 +53,10 @@ def check_uri_path
         errors.add(:url, 'the URL must have a path') if uri.blank? || uri.path.blank?
       end
 
+      def generate_secret_key
+        assign_attributes(secret_key: SecureRandom.urlsafe_base64(32))
```

Comment:
Is there a side-effect that we rely on here, or could it be simplified to `self.secret_key = SecureRandom.urlsafe_base64(32)`?

Reply:
No, there is no side effect here for the `secret_key` attribute in this case. Normally, I use the `assign_attributes` as it runs the "entire" lifecycle of attribute assignment. 

I'm fine with whatever you see fit here. If it's more common to use the `self.secret_key = ...`, I'm more than happy to change it.

Reply:
I just did a grep across the codebase, and we don't use assign_attributes in other places - it's it's always a direct assignment using `self.xxx = yyy`. If you could update that, it will be consistent with the rest of the codebase then. Thanks!